include/SDL3/SDL_mutex.h

```diff
@@ -30,6 +30,7 @@
 
 #include <SDL3/SDL_stdinc.h>
 #include <SDL3/SDL_error.h>
+#include <SDL3/SDL_thread.h>
```

Comment:
Should probably also include `SDL_atomic.h`, now that `SDL_AtomicInt` is used.

Reply:
SDL_thread.h already includes SDL_atomic.h, but it's probably better to be explicit.